### PR TITLE
ci: check pr deployment state before sync

### PR DESCRIPTION
**Pull request by mmpetarpeshev**

### CODE REVIEW

File: .github/workflows/frontend-staging-docker.yml [hunk 1]
```diff
@@ -84,6 +84,12 @@ jobs:
           ref: stg
           persist-credentials: false
           fetch-depth: 0
+          
+      - name: Chekc the initial PR deployment status before make a PR Sync.
```
Comment (thepiwo) [inline .github/workflows/frontend-staging-docker.yml:88]: same typo

File: .github/workflows/server-staging-docker.yml [hunk 2]
```diff
@@ -85,6 +85,12 @@ jobs:
           persist-credentials: false
           fetch-depth: 0
 
+      - name: Chekc the initial PR deployment status before make a PR Sync.
```
Comment (thepiwo) [inline .github/workflows/server-staging-docker.yml:88]: same typo

**Review by thepiwo — APPROVED** [review]

**Comment by thepiwo** [comment]
@mmpetarpeshev do you want to merge this?

**Comment by kenodressel** [comment]
Work was already done by @dincho :)

**Pull request closed without merging**